analyzer/src/main/resources/scripts/bundler_dependencies_metadata.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/ruby
```

Comment:
Commit message:

* "Add ruby script" -> "Add a Ruby script"
* "behaviour" -> "behavior" (we use American English)